docs/AppService.md

```diff
@@ -67,3 +67,138 @@ Run the Validation Test
+
+az container create -g $He_WebV_RG --image retaildevcrew/webvalidate:debug -o tsv --query name \
+-n ${He_Name}-webv-westeurope -l westeurope \
+--log-analytics-workspace $(eval $He_LogAnalytics_Id) --log-analytics-workspace-key $(eval $He_LogAnalytics_Key) \
+--command-line "dotnet ../webvalidate.dll --tag westeurope -l 10000 -s https://${He_Name}.azurewebsites.net -u https://raw.githubusercontent.com/retaildevcrews/${He_Repo}/master/TestFiles/ -f benchmark.json -r --json-log"
+
+az container create -g $He_WebV_RG --image retaildevcrew/webvalidate:debug -o tsv --query name \
+-n ${He_Name}-webv-southeastasia -l southeastasia \
+--log-analytics-workspace $(eval $He_LogAnalytics_Id) --log-analytics-workspace-key $(eval $He_LogAnalytics_Key) \
+--command-line "dotnet ../webvalidate.dll --tag southeastasia -l 10000 -s https://${He_Name}.azurewebsites.net -u https://raw.githubusercontent.com/retaildevcrews/${He_Repo}/master/TestFiles/ -f benchmark.json -r --json-log"
+
+```
+
+### Sample Queries
+
+Click on the 'Logs' item in the Log Analytics sidebar menu and run the `ContainerInstanceLog_CL` query to view all Azure Container Instance logs. Each log message consists of the following fields:
+
+- category - Type of request
+- path - Path to web API resource
+- tag - Tag associated when running Web Validate. For the smoke tests, this is the location where you created the ACI
+- statusCode - Status code from the test run
+- duration - Duration of the test
+- quartile - Based on the test duration
+- logType - Type of message, either request or summary
+- contentLength - Content length of test response
+- errorCount - Number of errors encountered running the test
+
+Refer to the Log Analytics Kusto Query Language (KQL) [overview](https://docs.microsoft.com/en-us/azure/data-explorer/kusto/query/) for further information on creating queries.
+
+```bash
+
+# Get 10 most recent log entries
+ContainerInstanceLog_CL | top 10 by TimeGenerated
+
+# Unpack all fields of Message object into columns
+ContainerInstanceLog_CL
+| extend jsonMessage = parsejson(Message)
+| extend category = tostring(jsonMessage.category),
+  path = tostring(jsonMessage.path),
+  tag = tostring(jsonMessage.tag),
+  code = toint(jsonMessage.statusCode),
+  duration = toint(jsonMessage.duration),
+  quartile = toint(jsonMessage.quartile),
+  logType = tostring(jsonMessage.logType),
+  length = toint(jsonMessage.contentLength),
+  errors = toint(jsonMessage.errorCount)
+
+# Failed or error tests
+ContainerInstanceLog_CL
+| extend jsonMessage = parsejson(Message)
+| extend errors = toint(jsonMessage.errorCount),
+  code = toint(jsonMessage.statusCode)
+| where code != 200 or errors > 0
+
+# Test runs exceeding 2000ms
+ContainerInstanceLog_CL
+| extend jsonMessage = parsejson(Message)
+| extend duration = toint(jsonMessage.duration)
+| where duration > 2000
+
+# Average duration of tests per category and location
```

Comment:
Wasn't sure of adding the `average quartile by category` query, seems pretty interchangeable with this.

Reply:
the quartile takes into account how long each request should take. since 2/3 of our requests are direct reads which should take about 20ms, the quartiles take that into the equation whereas > duration or avg duration don't